packages/graphback-cli/src/components/db.ts

```diff
@@ -13,57 +14,62 @@ const handleError = (err: { code: string; message: string; }): void => {
+      extensions: [graphbackConfigExtension, dbMigrationConfigExtension]
     });
-    const project = config.getProject(cliFlags.project || 'default')
-    const graphbackConfig = project.extension(extensionName);
+    const project = config.getProject(cliFlags.project || 'default');
+    const graphbackConfig = project.extension(graphbackExtension);
+    const dbMigrationConfig = project.extension(dbmigrationsExtension);
 
     if (!graphbackConfig) {
-      throw new Error(`You should provide a valid '${extensionName}' config to generate schema from data model`);
+      throw new Error(`You should provide a valid '${graphbackExtension}' config to generate schema from data model`);
+    }
+
+    if (!dbMigrationConfig) {
+      throw new Error(`You should provide a valid '${dbmigrationsExtension}' config to migrate schema`);
     }
 
     if (!graphbackConfig.model) {
-      throw new Error(`'${extensionName}' config missing 'model' value that is required`);
+      throw new Error(`'${graphbackExtension}' config missing 'model' value that is required`);
     }
 
-    const models = new GlobSync(`${graphbackConfig.model}/*.graphql`)
+    const models = new GlobSync(`${graphbackConfig.model}/*.graphql`);
 
     if (models.found.length === 0) {
-      logError(`No graphql file found inside ${process.cwd()}/model folder.`)
+      logError(`No graphql file found inside ${process.cwd()}/model folder.`);
       process.exit(0)
     }
 
-    if (!graphbackConfig.dbmigrations) {
-      throw new Error(`'${extensionName}' config missing 'dbmigrations' section that that is required. Please check example application for more information`);
-    }
+    // if (!graphbackConfig.dbmigrations) {
+    //   throw new Error(`'${graphbackExtension}' config missing 'dbmigrations' section that that is required. Please check example application for more information`);
+    // }
 
-    if (graphbackConfig.dbmigrations.client === 'sqlite3') {
-      logInfo(`SQLLite database will need to be recreated from scratch with every migration. Please manually remove/backup existing SQLLite database file before migration`)
-      await execa('touch', [graphbackConfig.dbmigrations.connection.filename])
+    if (dbMigrationConfig.client === 'sqlite3') {
+      logInfo(`SQLLite database will need to be recreated from scratch with every migration. Please manually remove/backup existing SQLLite database file before migration`);
+      await execa('touch', [dbMigrationConfig.connection.filename])
     }
 
     const migrateOptions: MigrateOptions = {
       //Do not perform delete operations on tables
       operationFilter: removeNonSafeOperationsFilter
-    }
+    };
 
-    const schemaText = loadSchema(graphbackConfig.model);
-    databaseOperations = await migrateDB(graphbackConfig.dbmigrations, schemaText, migrateOptions);
+    const schemaText = loadSchema(project.schema.toString());
+    databaseOperations = await migrateDB(dbMigrationConfig, schemaText, migrateOptions);
```

Comment:
migrateDb actually should allow both text and compiled schema. 

```
export async function migrateDB(
  config: Knex.Config,
  schema: string | GraphQLSchema,
  options: MigrateOptions = {},
): Promise<MigrationResults> {
```
This way api will be much simpler and we will not need to transform schema back and forth.
CC @craicoverflow